src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -21,6 +23,13 @@ import { SearchControls } from './search';
  */
 const SUB_CATEGORY_SHORT_SIZE = 10;
 
+// Organisms for which we display tabs
```

Comment:
Similar comment to my JSDoc earlier. JSDocs helps IDE make const more discoverable.